draft-irtf-cfrg-aead-limits.md

```diff
@@ -356,22 +365,33 @@ v <= (p * 2^127) / (l + 1)
 
 ## AEAD_CHACHA20_POLY1305
 
-The only known analysis for AEAD_CHACHA20_POLY1305 {{ChaCha20Poly1305Bounds}}
-combines the confidentiality and integrity limits into a single expression,
-covered below:
+The known single-user analyses for AEAD_CHACHA20_POLY1305 {{ChaCha20Poly1305-SU}},
+{{ChaCha20Poly1305-MU}} combine the confidentiality and integrity limits into a
+single expression, covered below. For this AEAD, n = 512, k = 256, and t = 128;
+the length l is the sum of AAD and plaintext (in blocks of 128 bits),
+see {{ChaCha20Poly1305-MU}}.
 
 <!-- I've got to say that this is a pretty unsatisfactory situation. -->
 
+<!--
+    In {{ChaCha20Poly1305-SU}}, L is |AAD| + |plaintext| + 1; the + 1 is one
+    block length encoding.
+    
+    From {{ChaCha20Poly1305-MU}} Theorem 4.1 / 3.4:
+      AE <= v * 2^25 * (l+1) / 2^t
+    where t = 128.
+    (NB: The bound component "c * l" (for c = 3*2^24) is upper-bounding
+    2^25 * (l+1) for the worst case l = |AAD|+|m| = 2; cf. Theorem 3.4.)
```

Comment:
> Instead of assuming the ChaCha block function to be a good PRF (resulting in a PRF advantage term), the multi-user analysis idealizes the permutation (hence that term vanishes).

Gotcha 👍  I must have missed that in the paper.